server/src/main/java/com/objectcomputing/checkins/services/template_question/TemplateQuestionServicesImpl.java

```diff
@@ -0,0 +1,147 @@
+package com.objectcomputing.checkins.services.template_question;
+
+import com.objectcomputing.checkins.exceptions.BadArgException;
+import com.objectcomputing.checkins.exceptions.NotFoundException;
+import com.objectcomputing.checkins.exceptions.PermissionException;
+import com.objectcomputing.checkins.services.feedback_template.FeedbackTemplate;
+import com.objectcomputing.checkins.services.feedback_template.FeedbackTemplateServices;
+import com.objectcomputing.checkins.services.memberprofile.MemberProfileServices;
+import com.objectcomputing.checkins.services.memberprofile.currentuser.CurrentUserServices;
+import com.objectcomputing.checkins.util.Util;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Singleton;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Singleton
+public class TemplateQuestionServicesImpl implements TemplateQuestionServices {
+
+    private final TemplateQuestionRepository templateQuestionRepository;
+    private final CurrentUserServices currentUserServices;
+    private final FeedbackTemplateServices feedbackTemplateServices;
+    private static final Logger LOG = LoggerFactory.getLogger(TemplateQuestionServicesImpl.class);
+
+    public TemplateQuestionServicesImpl(TemplateQuestionRepository templateQuestionRepository,
+                                        CurrentUserServices currentUserServices,
+                                        MemberProfileServices memberProfileServices,
+                                        FeedbackTemplateServices feedbackTemplateServices) {
+        this.templateQuestionRepository = templateQuestionRepository;
+        this.currentUserServices = currentUserServices;
+        this.feedbackTemplateServices = feedbackTemplateServices;
+    }
+
+    @Override
+    public TemplateQuestion update(TemplateQuestion templateQuestion) {
+        FeedbackTemplate feedbackTemplate;
+        TemplateQuestion oldTemplateQuestion = null;
+
+        LOG.info("Template question {}:", templateQuestion);
+        if (templateQuestion.getId() != null) {
+             oldTemplateQuestion = getById(templateQuestion.getId());
+             LOG.info("Old template question: {} " + oldTemplateQuestion.toString());
```

Comment:
same here. is there a need or they were added for debugging purpose ?

Reply:
Just debugging purposes. They have been removed.